src/api/dino.ts

```diff
@@ -112,11 +112,12 @@ export class Dino implements IDino {
     // since container is already created and bound to express instance
     /**
      * Binds the dino instance to express app
-     * @Throws Error(Errors.routerNotRegistered) 
+     * @Throws Error(Errors.routerNotRegistered)
      * @Throws Error(Errors.dinoAlreadyBinded)
      */
     bind(): void {
         if (this.isBinded) throw new Error(Errors.dinoAlreadyBinded);
+        if (!this.appContainer.baseUri) throw new Error(Errors.baseUriInvalid);
```

Comment:
Could you please update to `DataUtility.isUndefinedOrNull `over negation to have consitent coding guidelines across the codebase